src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -34,116 +29,13 @@ public ConsoleExporter(ConsoleExporterOptions options)
-                            this.WriteLine($"    {activityEvent.Name} [{activityEvent.Timestamp}]");
-                            foreach (var attribute in activityEvent.Tags)
-                            {
-                                this.WriteLine($"        {attribute.Key}: {attribute.Value}");
-                            }
-                        }
-                    }
-
-                    if (activity.Baggage.Any())
-                    {
-                        this.WriteLine("Activity.Baggage:");
-                        foreach (var baggage in activity.Baggage)
-                        {
-                            this.WriteLine($"    {baggage.Key}: {baggage.Value}");
-                        }
-                    }
-
-                    var resource = this.ParentProvider.GetResource();
-                    if (resource != Resource.Empty)
-                    {
-                        this.WriteLine("Resource associated with Activity:");
-                        foreach (var resourceAttribute in resource.Attributes)
-                        {
-                            this.WriteLine($"    {resourceAttribute.Key}: {resourceAttribute.Value}");
-                        }
-                    }
-
-                    this.WriteLine(string.Empty);
-                }
-            }
-#if NET461 || NETSTANDARD2_0
-            else if (typeof(T) == typeof(LogRecord))
-            {
-                var rightPaddingLength = 30;
-                foreach (var item in batch)
-                {
-                    var logRecord = item as LogRecord;
-                    this.WriteLine($"{"LogRecord.TraceId:".PadRight(rightPaddingLength)}{logRecord.TraceId}");
-                    this.WriteLine($"{"LogRecord.SpanId:".PadRight(rightPaddingLength)}{logRecord.SpanId}");
-                    this.WriteLine($"{"LogRecord.Timestamp:".PadRight(rightPaddingLength)}{logRecord.Timestamp:yyyy-MM-ddTHH:mm:ss.fffffffZ}");
-                    this.WriteLine($"{"LogRecord.EventId:".PadRight(rightPaddingLength)}{logRecord.EventId}");
-                    this.WriteLine($"{"LogRecord.CategoryName:".PadRight(rightPaddingLength)}{logRecord.CategoryName}");
-                    this.WriteLine($"{"LogRecord.LogLevel:".PadRight(rightPaddingLength)}{logRecord.LogLevel}");
-                    this.WriteLine($"{"LogRecord.TraceFlags:".PadRight(rightPaddingLength)}{logRecord.TraceFlags}");
-                    this.WriteLine($"{"LogRecord.State:".PadRight(rightPaddingLength)}{logRecord.State}");
-                    if (logRecord.Exception is { })
-                    {
-                        this.WriteLine($"{"LogRecord.Exception:".PadRight(rightPaddingLength)}{logRecord.Exception?.Message}");
-                    }
-
-                    this.WriteLine(string.Empty);
-                }
-            }
-#endif
-
-            return ExportResult.Success;
-        }
+        public abstract override ExportResult Export(in Batch<T> batch);
```

Comment:
I think you can leave this line out and it should work fine. `Export` is already `abstract` on base so the supers will need to implement it, `ConsoleExporter` doesn't have to because it is also `abstract`.

Reply:
Fixed this.